jovial.zsh-theme

```diff
@@ -456,7 +456,11 @@ typeset -gA jovial_affix_lengths=()
 }
 
 @jov.set-venv-info() {
-    if [[ -z ${VIRTUAL_ENV} ]]; then
+    if [ ! -z $CONDA_DEFAULT_ENV ]; then
+        export VIRTUAL_ENV=$CONDA_DEFAULT_ENV
```

Comment:
Setting `export VIRTUAL_ENV=$CONDA_DEFAULT_ENV` seems like a bad idea because the two variables do not sync with each other. When deactivating with Conda, the `VIRTUAL_ENV` variable is not reset.

Reply:
Currently, theme move the condaenv in top of the line, like the following screenshot:
![CleanShot 2024-07-15 at 19 31 58](https://github.com/user-attachments/assets/08ac2e71-2eee-46bb-a498-5c891c422af7)

Reply:
II think the (base) env shoud be placed in the right of the arrow